src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -740,7 +757,6 @@ private static void PartialMethodConstraintsChecks(SourceOrdinaryMethodSymbol de
             {
                 var typeParameter1 = typeParameters1[i];
                 var typeParameter2 = typeParameters2[i];
```

Comment:
Consider restoring the deleted blank line.

```suggestion
                var typeParameter2 = typeParameters2[i];

```